src/test/java/uk/co/ramp/covid/simulation/CovidTest.java

```diff
@@ -14,37 +14,37 @@
 
     //Test that a pensioner steps through the infection from latent to death
     @Test
-    public void testStepInfection() throws IOException {
+    public void testStepInfectionSymptomatic() throws IOException {
         RNG.seed(321);
         //Use the test file with a mortality rate of 100
         ParameterReader.readParametersFromFile("src/test/resources/test_params.json");
         CStatus cStatus = null;
         Person pensioner = new Pensioner();
         Covid virus = new Covid(pensioner);
-
+        virus.forceSymptomatic(true);
         //Test that the person is latent at the end of the latent period
-        int latentPeriod = virus.getLatentPeriod() * 24 - 1;
+        double latentPeriod = virus.getLatentPeriod() - 1;
         for (int i = 0; i < latentPeriod; i++) {
-            cStatus = virus.stepInfection();
+
+        	cStatus = virus.stepInfection();
         }
         assertEquals(CStatus.LATENT, cStatus);
 
-        //Test that the person becomes asymptomatic in the next time period
-        cStatus = virus.stepInfection();
-        assertEquals(CStatus.ASYMPTOMATIC, cStatus);
-
         //Test that the person becomes phase1 after the asymptomatic period
-        double asymptomaticPeriod = virus.getAsymptomaticPeriod() * 24;
-        for (int i = 0; i < asymptomaticPeriod; i++) {
+        double p1Period = virus.getP1();
+
+        for (int i = 0; i < p1Period; i++) {
             cStatus = virus.stepInfection();
         }
         assertEquals(CStatus.PHASE1, cStatus);
+        assertEquals(true, virus.getIsSymptomatic());
```

Comment:
There's an "assertTrue" you can use for this

Reply:
Done